FreshRSS/FreshRSS
# New option: Wrap titles

Closes #4428
Closes #3740
Ref. #3997

Changes proposed in this pull request:

- add an option to wrap long titles in normal view and global view
- options for big screens ("Desktop") and small screens ("mobile devices")

![grafik](https://user-images.githubusercontent.com/1645099/188701673-b321b195-3fe4-42f1-b147-7ae6e9e3949b.png)


How to test the feature manually:

1. go to `configuration` -> `reading`
2. enable/disable the checkboxes
3. check the titles in normal view or global view

Pull request checklist:
